scripts/release-update-tilt-docs-repo.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Updates the Tilt docs repo with the latest version info.
+#
+# Usage:
+# scripts/update-docs-tilt-repo.sh $VERSION
+# where VERSION is of the form 0.1.0
+
+set -e
+
+if [[ "$GITHUB_TOKEN" == "" ]]; then
+    echo "Missing GITHUB_TOKEN"
+    exit 1
+fi
+
+VERSION=${1//v/}
+VERSION_PATTERN="[0-9]+\.[0-9]+\.[0-9]+"
+if ! [[ $VERSION =~ $VERSION_PATTERN ]]; then
+    echo "Version did not match expected pattern. Actual: $VERSION"
+    exit 1
+fi
+
+DIR=$(dirname "$0")
+cd "$DIR/.."
+
+ROOT=$(mktemp -d)
+git clone https://tilt-releaser:"$GITHUB_TOKEN"@github.com/tilt-dev/tilt.build "$ROOT"
+
+set -x
+go run ./cmd/tilt/main.go dump cli-docs --dir="$ROOT/docs/cli"
+cd "$ROOT"
+sed -i -e "s/asdf install tilt .*/asdf install tilt $VERSION/" docs/install.md
```

Comment:
In BSD sed, this will create a file named "install.md-e". Maybe we should make a common.sh that we `source` in our other scripts and add a replace function there.

Reply:
orrrr this is running in a linux container, so that's irrelevant? yay!

Reply:
ya, i'm just going to make this run in an alpine/git container